python/lsst/ts/dream/csc/dream_csc.py

```diff
@@ -485,9 +494,16 @@ async def weather_and_status_loop(self) -> None:
                             weatherFlags |= Weather.WeatherBad | Weather.WindBad
 
                     if use_precipitation:
-                        precipitation = await ess_remote.evt_precipitation.aget(
-                            timeout=SAL_TIMEOUT
-                        )
+                        try:
+                            precipitation = await ess_remote.evt_precipitation.aget(
+                                timeout=SAL_TIMEOUT
+                            )
+                        except TimeoutError:
+                            precipitation = None
+                            self.log.error(
```

Comment:
same comment as above about being explicit that this is a timeout and about this being at error level.